deployment/src/main/java/io/quarkiverse/kerberos/deployment/devservices/DevServicesConfig.java

```diff
@@ -0,0 +1,87 @@
+package io.quarkiverse.kerberos.deployment.devservices;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class DevServicesConfig {
+
+    /**
+     * If DevServices has been explicitly enabled or disabled.
+     * <p>
+     * When DevServices is enabled Quarkus will attempt to automatically configure and start
+     * Kerberos when running in Dev or Test mode and when Docker is running.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled = true;
+
+    /**
+     * The container image name to use, for container based DevServices providers.
+     * See https://github.com/kerberos-io/kerberos-docker.
+     */
+    @ConfigItem(defaultValue = "gcavalcante8808/krb5-server")
```

Comment:
This does not look like any sort of official image. I don't think we should just be running images from peoples personal accounts.

Reply:
Unfortunately, there is no "official" image. It's the most used image. I would like to avoid having to build an image ourselves.